Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.yml

```diff
@@ -1571,11 +1571,13 @@ script:
       - PATCH
       - DELETE
       required: true
-    - description: the API path starting with forward slash (/).
+    - description: the API path (without a leading forward slash '/').
```

Comment:
This is a very problematic change since its considered as breaking backwards competability. 
If other customers use this command in a playbook or any other custom made content use case, knowing that the path parameter must start with a starting slash as documented, and you want to change it to non leading forward slash that will break the already made content. 
We do not allow such changes to xsoar supported packs.

Reply:
There's a misunderstanding here. This change is only a correction of the path argument description. The previously description said that the path argument needs to start with a forward slash. But this was actually wrong and the command DOES NOT work if the path has a leading forward slash.
As you can see in the base_url definition here (line 2978-2989):
```python
    if version:
        api = f'/api/now/{version}/'
        sc_api = f'/api/sn_sc/{version}/'
        cr_api = f'/api/sn_chg_rest/{version}/'
    else:
        api = '/api/now/'
        sc_api = '/api/sn_sc/'
        cr_api = '/api/sn_chg_rest/'
    server_url = params.get('url')
    sc_server_url = f'{get_server_url(server_url)}{sc_api}'
    cr_server_url = f'{get_server_url(server_url)}{cr_api}'
    server_url = f'{get_server_url(server_url)}{api}'
``` 
the server url always ends with a forward `/`. So if the path argument starts with a forward slash, it would lead to something like `https://xy.servicenow.com/api/now//my/path` (see double slashes `//` between `now` and `my`) which leads to an error. This is the case in the actual version of the ServiceNowv2 integration therefore we corrected the description so the end users can save their debugging time (because the resulting error message doesn't directly show the problem with two forward slashes).

**NO functionality was changed, only the description was fixed. Therefore no breaking change.**

Reply:
For the context, this is the difference of a leading forward slash (as it must have according to the previous description) VS without a leading forward slash (as in the corrected description) in the current ServiceNowv2 Integration (without the changes of this PR):
![image](https://github.com/demisto/content/assets/5670794/d1703927-92eb-487d-8a00-f357e031b4f6)

Reply:
please revert